okhttp/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -71,6 +72,7 @@
 import static org.junit.Assert.fail;
 import static org.junit.Assume.assumeThat;
 
+@Flaky // STDOUT logging enabled for test
```

Comment:
This just makes sense while making any changes to the EventListenerTest.  Is there anywhere this creates too much noise?

Reply:
Doesn’t bother me.